package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-google-places-autocomplete",
-  "version": "2.1.1",
+  "version": "2.1.2",
```

Comment:
can you revert this. Versions are automatically bumped for each release.

Reply:
done